docs/components/motor/gpio/_index.md

```diff
@@ -118,40 +118,32 @@ The following attributes are available for `gpio` motors:
 
 Refer to your motor and motor driver data sheets for specifics.
 
-{{% alert title="Caution" color="caution" %}}
+## `pins`
 
-The attribute `max_rpm`is not required or available for [encoded](/components/motor/gpio/encoded-motor/) `gpio` motors.
+There are three common ways for your computer to communicate with a brushed DC motor driver chip.
+**Your motor driver data sheet will specify which one to use.**
 
-{{% /alert %}}
+- PWM/DIR: Two pins: One digital input (such as a GPIO pin) sends a [pulse width modulation (PWM)](https://en.wikipedia.org/wiki/Pulse-width_modulation) signal to the driver to control speed while another digital input sends a high or low signal to control the direction.
+  - Configure `pwm` and `dir`.
+- In1/In2 (also known as A/B): Two pins: One digital input is set to high and another set to low turns the motor in one direction and vice versa, while speed is controlled with PWM through one or both pins.

```

Comment:
```suggestion
- In1/In2: Two pins on the motor, often labeled In1 and In2 or A/B, connected to two GPIO pins on the board. One pin is set to high (active, >0), and another is set to low (inactive, 0V) to turn the motor in different directions depending on which pin is high. Speed is controlled with PWM through one or both pins.
```
Same suggestion format above--- No matter what I think nesting the colons should be edited!